examples/seq2seq/finetune.sh

```diff
@@ -1,11 +1,24 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # the proper usage is documented in the README, you need to specify data_dir, output_dir and model_name_or_path
-# run ./finetune.sh --help to see all the possible options
-python finetune.py \
+# run ./builtin_trainer/finetune.sh --help to see all the possible options
```

Comment:
what about people using the original PL-based script? are you saying it's no more?

Reply:
The original PL-based script is moved to the research folder seq2seq, the example focuses on Trainer only.

Reply:
This might be confusing for people who will try to run this script post merge - perhaps leaving a note where the original moved to? The confusion is that the old name is re-used for a totally different script.

Reply:
I could be wrong, but I got the impression from Sam that this was a very widely used script - I can't tell, perhaps it's not. But if it is a deprecation-like cycle - at least leaving a hint is a nice gesture towards users.